### PR TITLE
confirmation dialog added for job creation 

### PR DESCRIPTION
**Pull request by horaanchal**

### CODE REVIEW

File: api/routes/index.js [hunk 1]
```diff
@@ -74,10 +56,10 @@ module.exports = (app) => {
...
   );
 
-  app.post("/api/checkvalidemployee", (req, res) => 
+  app.post("/api/checkvalidemployee", (req, res) =>
     controller.login.checkValidEmployee(req, res)
   );
   // app.post('/api/candidate',upload.single('file'), (req,res)=> controller.candidate.uploadDetails(req,res));
   //login route
-  app.post("/api/checkvalidemployee",controller.login.checkValidEmployee);
+  app.post("/api/checkvalidemployee", controller.login.checkValidEmployee);
```
Comment (sonalichawla16) [inline api/routes/index.js:64]: maintain casing please.

File: api/storage/csv/employee.csv [hunk 2]
```diff
@@ -2,4 +2,8 @@ employeeId,email,name,designation,role
 100,senseihimanshu@gmail.com,Himanshu Sharma,intern,interviewer
 101,nangchetnamaw@gmail.com,Chetna Mongmaw,intern,interviewer
 105,deepak.yadav@gmail.com,Deepak Yadav,intern,interviewer
-106,faizan.akhtar@gmail.com,Faizan Akhtar,intern,interviewer
+106,sankalp.shastri@gmail.com,Sankalp Shastri,intern,interviewer
+107,sonali.chawla@gmail.com,Sonali Chawla,intern,interviewer
+108,lord.rama@gmail.com,Lord Rama,manager,superuser
```
Comment (sonalichawla16) [inline api/storage/csv/employee.csv:7]: please remove this file

File: web/src/app/employee/components/employee-form/employee-form.component.html [hunk 3]
```diff
@@ -36,9 +35,9 @@ <h5 class="modal-title" id="exampleModalLabel">{{ (formType | titlecase) + ' Emp
           <select name="role" [ngModel]="employee?.role" required class="form-control" id="role" #role="ngModel"
             [ngClass]="{ 'is-invalid': role.dirty && role.invalid }">
             <option [value]="null" disabled>Select Role</option>
+            <option value="superuser">Superuser</option>
```
Comment (sonalichawla16) [inline web/src/app/employee/components/employee-form/employee-form.component.html:38]: Maintain casing

File: web/src/app/employee/models/employee.interface.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
 type Designation = 'intern' | 'consultant1' | 'consultant2' | 'associate1' | 'associate2' | 'manager';
-type Role = 'admin' | 'hr' | 'interviewer';
+type Role = 'superuser' | 'admin' | 'employee';
```
Comment (sonalichawla16) [inline web/src/app/employee/models/employee.interface.ts:2]: superUser

Comment (sonalichawla16) [reply web/src/app/employee/models/employee.interface.ts:2]: Super User

File: api/routes/index.js (hunk 1 above)
Comment (senseihimanshu) [reply api/routes/index.js:64]: Done!

File: api/storage/csv/employee.csv (hunk 2 above)
Comment (senseihimanshu) [reply api/storage/csv/employee.csv:7]: Done!

File: web/src/app/employee/components/employee-form/employee-form.component.html (hunk 3 above)
Comment (senseihimanshu) [reply web/src/app/employee/components/employee-form/employee-form.component.html:38]: Done

File: web/src/app/employee/models/employee.interface.ts (hunk 4 above)
Comment (senseihimanshu) [reply web/src/app/employee/models/employee.interface.ts:2]: We are maintaining enums by this convention only!

**Pull request merged**